libs/wingc/src/type_check.rs

```diff
@@ -250,7 +250,7 @@ pub enum Type {
 	Enum(Enum),
 }
 
-pub const CLASS_INIT_NAME: &'static str = "init";
+pub const CLASS_INIT_NAME: &'static str = "init"; // TODO: change to "new"??
```

Comment:
I think this should be something like `$new` so that we don't take up this symbol (to get rid of https://github.com/winglang/wing/issues/5441)

Reply:
Removed this TODO, since I see we have an issue for it.